json.go

```diff
@@ -216,13 +231,30 @@ func (json *JSONOverlapsExpression) Build(builder clause.Builder) {
 	if stmt, ok := builder.(*gorm.Statement); ok {
 		switch stmt.Dialector.Name() {
 		case "mysql":
-			builder.WriteString("JSON_OVERLAPS(" + stmt.Quote(json.column) + ",")
+			builder.WriteString("JSON_OVERLAPS(")
+			json.column.Build(builder)
+			builder.WriteString(",")
 			builder.AddVar(stmt, json.val)
 			builder.WriteString(")")
 		}
 	}
 }
 
+type columnExpression string
+
+func Column(col string) columnExpression {
+	return columnExpression(col)
+}
+
+func (col columnExpression) Build(builder clause.Builder) {
+	if stmt, ok := builder.(*gorm.Statement); ok {
+		switch stmt.Dialector.Name() {
+		case "mysql":
```

Comment:
the Column should be able to works for all drivers?

Reply:
I'm not sure, I think it can support sqlite, mysql, postgres, not test.

Reply:
Update: support sqlite, mysql, postgres.
Rebase code with master.